### ISSUE TITLE
Abnormal Reg loss

### ISSUE THREAD

**Issue by htx-DeepLearning** [zylo117/Yet-Another-EfficientDet-Pytorch#67]
Thanks for your great job! 
The question is when I fine-tune d5 model on my own dataset, the cls loss decreases normally while the reg loss does not decrease 
Could you help me?@zylo117 
I'm sure the dataset is right because I have trained it on other model and get a good result.
![微信截图_20200414114652](https://user-images.githubusercontent.com/51436385/79184130-e64ef980-7e45-11ea-8c06-4d4a5e5e34bd.png)
![微信截图_20200414114641](https://user-images.githubusercontent.com/51436385/79184134-e7802680-7e45-11ea-8456-5c645a780f11.png)

**Comment by zylo117** [comment]
what is your gt format? It should be [x1,y1,w,h]. The program then changes it into [x1, y1, x2, y2]

**Comment by htx-DeepLearning** [comment]
@zylo117 Thanks for you reply. The gt format of my own dataset is  [x1,y1,w,h]. I have examined my dataset through pycocotools api, and it can correctly load annotations.

**Comment by zylo117** [comment]
is the shape ratio very different from coco's? if so, the cls loss will be faking low

**Comment by zylo117** [comment]
there's a bug in reg loss func, so sorry for the trouble. please pull the latest code and try again
